modules/cloudbuild/cloudbuild_builder/README.md

```diff
@@ -0,0 +1,14 @@
+# [Terraform](https://www.terraform.io/docs) cloud builder
```

Comment:
I suggest moving the link from this header to the body of the next section.

Reply:
I have tidied this up, was derived from the community terraform builder.